<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frameDirs">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>4</y>
      <width>481</width>
      <height>89</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="buttonCompare">
     <property name="geometry">
      <rect>
       <x>388</x>
       <y>56</y>
       <width>93</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Compare</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="editBaseline">
     <property name="geometry">
      <rect>
       <x>68</x>
       <y>4</y>
       <width>320</width>
       <height>24</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
     <property name="documentTitle">
      <string/>
     </property>
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>8</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Baseline:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>36</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Target:</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="editTarget">
     <property name="geometry">
      <rect>
       <x>68</x>
       <y>32</y>
       <width>320</width>
       <height>24</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonBrowseBaseline">
     <property name="geometry">
      <rect>
       <x>388</x>
       <y>0</y>
       <width>93</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonBrowseTarget">
     <property name="geometry">
      <rect>
       <x>388</x>
       <y>28</y>
       <width>93</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse...</string>
     </property>
    </widget>
   </widget>
   <widget class="QTreeView" name="treeViewCompare">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>96</y>
      <width>481</width>
      <height>489</height>
     </rect>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="autoScroll">
     <bool>true</bool>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="animated">
     <bool>true</bool>
    </property>
    <property name="headerHidden">
     <bool>false</bool>
    </property>
    <attribute name="headerVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="headerMinimumSectionSize">
     <number>30</number>
    </attribute>
    <attribute name="headerStretchLastSection">
     <bool>false</bool>
    </attribute>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>488</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBrowseBaseline</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>onBrowseBaselinePressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>438</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onBrowseBaselinePressed()</slot>
 </slots>
</ui>
